### DISCUSSION TITLE
Is there any way to prefixes every href i made ?

### DISCUSSION THREAD

**Discussion by kafiediara** [laravel/framework#37040] [Q&A]
So basically, i already hosted my web and it has some prefixes like testmyweb.id/dev/public to open it. The problem is, whenever i open a page that runs API ,  the url back to its base (ex : testmyweb.id/showRoom) , so the problem occurs and the page does not show up. Is there any way to have a prefix on every href so any link i click will have /dev/public in front of it? Thank you!. i tried routes prefixes, but instead it prefix the route not the href.

**Comment by danjohnson95** [comment]
Sounds like you could benefit from using the `<base>` html tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
